deploy/deployment.yaml

```diff
@@ -1,144 +1,156 @@
-              persistentVolumeClaim:
-                  claimName: caesar-shared-static
-            - configMap:
-                  defaultMode: 256
-                  name: nginx-config
-                  optional: false
-              name: nginx-config-volume
+              optional: false
+        - name: OAUTH_ALLOWED_DOMAINS
+          valueFrom:
+            secretKeyRef:
+              key: OAUTH_ALLOWED_DOMAINS
+              name: caesarapp-server
+              optional: false
+        - name: GOOGLE_ID
+          valueFrom:
+            secretKeyRef:
+              key: GOOGLE_ID
+              name: caesarapp-server
+              optional: false
+        - name: GOOGLE_SECRET
+          valueFrom:
+            secretKeyRef:
+              key: GOOGLE_SECRET
+              name: caesarapp-server
+              optional: false
+        - name: JWT_PASSPHRASE
+          valueFrom:
+            secretKeyRef:
+              key: JWT_PASSPHRASE
+              name: caesarapp-server
+              optional: false
+        - name: SUPERADMIN_PASS
+          valueFrom:
+            secretKeyRef:
+              key: SUPERADMIN_PASS
+              name: caesarapp-server
+              optional: false
+        - name: TWO_FACTOR_SERVER_NAME
+          valueFrom:
+            secretKeyRef:
+              key: TWO_FACTOR_SERVER_NAME
+              name: caesarapp-server
+              optional: false
+        - name: ALLOW_FRONT_REDIRECT_PATTERN
+          valueFrom:
+            secretKeyRef:
+              key: ALLOW_FRONT_REDIRECT_PATTERN
+              name: caesarapp-server
+              optional: false
+        - name: APP_SECRET
+          valueFrom:
+            secretKeyRef:
+              key: APP_SECRET
+              name: caesarapp-server
+              optional: false
+        - name: DATABASE_URL
+          valueFrom:
+            secretKeyRef:
+              key: DATABASE_URL
+              name: caesarapp-server
+              optional: false
+        - name: MAILER_URL
+          valueFrom:
+            secretKeyRef:
+              key: MAILER_URL
+              name: caesarapp-server
+              optional: false
+        image: caesarteam/caesar-server:${CICD_GIT_BRANCH}
+        imagePullPolicy: Always
+        name: caesarapp-server
+        ports:
+        - containerPort: 9000
+          name: 9000tcp00
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /var/www/html/public
+          name: caesar-shared-static
+      - image: nginx:alpine
+        imagePullPolicy: IfNotPresent
+        name: nginx
+        ports:
+        - containerPort: 80
+          name: 80tcp00
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /etc/nginx/nginx.conf
+          name: nginx-config-volume
+          subPath: nginx.conf
+        - mountPath: /var/www/html/public
+          name: caesar-shared-static
+      dnsPolicy: ClusterFirst
+      imagePullSecrets:
+      - name: dockerhub
```

Comment:
Why do we have so many duplicates here?

Reply:
@beshkenadze What are duplicates?

Reply:
@dyudin4xxi this one 
https://github.com/caesar-team/caesar-server/blob/6c7f83f27de9ea5f8a9ea00474e7b029b9603308/deploy/deployment.yaml#L124

Reply:
@beshkenadze For each of the three images launched by the docker, access to the docker hub is prescribed; if one of them is removed, then everything will break